modules/proc_use-api.adoc

```diff
@@ -104,6 +101,82 @@ Use the following procedure to create an OAuth access token.
 This is the only opportunity to copy and save the access token. It cannot be reobtained after leaving this page. 
 ====
 
+[id="reassigning-oauth-access-token"]
+== Reassigning an OAuth access token
+
+Organization administrators can take control of OAuth API tokens created by other users and re-assign those tokens to a different user. This helps audit logs properly reflect which member of an organization used, or is using, a token after reassignment.
+
+[NOTE]
+====
+The following procedure only works on the current {productname} UI. It is not currently implemented in the {productname} v2 UI.
+====
+
+.Prerequisites
+
+* You have created and assigned an OAuth token to a member in your organization.
+* You have administrative privileges for the registry. Superusers cannot reassign OAUth API tokens unless they have administrative privileges on the organization in which the application belongs to.
```

Comment:
This would be more accurate as `In order to assign the OAuth API token the user must of organization admin privileges`